packages/react-native-editor/__device-tests__/helpers/test-data.js

```diff
@@ -141,3 +141,27 @@ exports.fileBlockPlaceholder = `<!-- wp:file {"id":3,"href":"https://wordpress.o
 exports.audioBlockPlaceholder = `<!-- wp:audio {"id":5} -->
 <figure class="wp-block-audio"><audio controls src="https://cldup.com/59IrU0WJtq.mp3"></audio></figure>
 <!-- /wp:audio -->`;
+
+exports.headingExpectedHTML = `<!-- wp:heading -->
+<h2></h2>
+<!-- /wp:heading -->`;
+
+exports.separatorExpectedHTML = `<!-- wp:separator -->
+<hr class="wp-block-separator has-alpha-channel-opacity"/>
+<!-- /wp:separator -->`;
+
+exports.listExpectedHTML = `<!-- wp:list -->
+<ul><li></li></ul>
+<!-- /wp:list -->`;
+
+exports.imageExpectedHTML = `<!-- wp:image -->
+<figure class="wp-block-image"><img alt=""/></figure>
+<!-- /wp:image -->`;
+
+exports.moreExpectedHTML = `<!-- wp:more -->
+<!--more-->
+<!-- /wp:more -->`;
+
+exports.paragraphExpectedHTML = `<!-- wp:paragraph -->
+<p></p>
+<!-- /wp:paragraph -->`;
```

Comment:
I'm wondering if instead of naming these variables with `expected` and `HTML` words, we could use `empty` as they are the empty versions of blocks (i.e. `headingExpectedHTML` => `headerBlockEmpty`), wdyt?